huaweicloud/services/acceptance/cdn/data_source_huaweicloud_cdn_domain_certificates_test.go

```diff
@@ -0,0 +1,64 @@
+package cdn
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceCDNDomainCertificates_basic(t *testing.T) {
```

Comment:
Add a note: Before executing this use case, please create several pieces of data first.